sqrl-tools/sqrl-config/src/main/java/com/datasqrl/config/ConnectorFactoryFactoryImpl.java

```diff
@@ -148,4 +170,14 @@ private ConnectorFactory createJdbcConnectorFactory(ConnectorConf connectorConf)
       return builder.build();
     };
   }
+
+  private ConnectorFactory createBlackHoleConnectorFactory() {
```

Comment:
used?

Reply:
No, that remained from an intermediate solution where I routed LogMethod.NONEs to blackhole.